### PR TITLE
Team Minutes for Feb 27

### LINKED ISSUE #262: Team Meeting Minutes for Feb 27
**Describe what needs to get done**
- take minutes during the meeting on Feb 13

**Describe the solution you'd like**
- markdown file in docs>meetings>team

### PR DESCRIPTION
**Pull request by vnajack**
- minutes from our meeting today
- closes #262

### CODE REVIEW

File: docs/meetings/team/2023_02_27.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+### Issues this week
+
+- We want to figure out why the servers are performing worse than our local machines (Matt)
+  - Need to investigate if our API is opening threads appropriately
+- Update Usability Task List (Veronica)
+  - repeat scenarios only if there's time
+  - test with server rather than local machine
+  - update instructions for peer testing to fill out Google Form at the same time
+- Google Form for feedback (Team)
```
Comment (edouarde1) [inline docs/meetings/team/2023_02_27.md:62]: Can help out with this! This time around will probably not need to focus on edge cases and relate the questions towards performance and quality ?

File: docs/meetings/team/2023_02_27.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  - keep track of parameters for each scenario
+  - list of questions for feedback
+- Review current documentation to ensure it's accurate and updated, then convert to GitHub Pages
+  - Currently, we have our README, separate API documentation, and separate tooling documentation
+- Video demo: let's screen record and voice over again.
```
Comment (edouarde1) [inline docs/meetings/team/2023_02_27.md:67]: another vid demo! haha, happy to help out with the editing. Should be free this week to help record

File: docs/meetings/team/2023_02_27.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  - It's a busy week, so it might only be done with a couple group members instead of everyone
+
```
Comment (edouarde1) [inline docs/meetings/team/2023_02_27.md:69]: I will work on getting Rhubarb to work with the CI/CD script in git this week

File: docs/meetings/team/2023_02_27.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+## 🤝 Client Meeting
+
+- Show off poses and high-resolution
+- Check on the asset pose request
+- Performance report: Show our results in a table for audio files of different lengths (10s, 30s, 1:00, 1:30, etc.)
```
Comment (edouarde1) [inline docs/meetings/team/2023_02_27.md:74]: After we present the performance report, I hope they give us some more guidance on expectations and/or what our next steps to seeing how our system will perform on their services.

**Review by edouarde1 — COMMENTED** [review]
Thank you for these minutes! 

I added some comments on some things I can help out with this week

File: docs/meetings/team/2023_02_27.md (hunk 1 above)
Comment (vnajack) [reply docs/meetings/team/2023_02_27.md:62]: I'll make the Google Form today and share the link in Discord!

**Review by edouarde1 — APPROVED** [review]
forgot to approve... oops

thank you!!

File: docs/meetings/team/2023_02_27.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+## 📢 Announcements
+
+- Welcome back from Reading Week!
+- Due this week:
+  - GitStats Week 21 due this Friday
+- Upcoming
+  - Usability Task List due March 7
+  - Video Demo #2 due March 14
+  - Peer Testing II sessions are on March 15 and 17
```
Comment (LoganParker) [inline docs/meetings/team/2023_02_27.md:35]: Two weeks from now!

File: docs/meetings/team/2023_02_27.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+## 📅 Last 2 Weeks (Restrospective)
+
+- We're nearing the end which feels good, and we feel good about our progress
+- Veronica updated the usability tasks, but we should go over it together as a group!
```
Comment (LoganParker) [inline docs/meetings/team/2023_02_27.md:40]: usability tasks look great, thanks Veronica!

File: docs/meetings/team/2023_02_27.md (hunk 2 above)
Comment (LoganParker) [reply docs/meetings/team/2023_02_27.md:67]: @edouarde1 We can maybe get this done over the weekend!

File: docs/meetings/team/2023_02_27.md (hunk 4 above)
Comment (LoganParker) [reply docs/meetings/team/2023_02_27.md:74]: This is definitely something we should bring up with them in the next meeting!

File: docs/meetings/team/2023_02_27.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+***Attendence Comments***
+
+- Logan is in Hawaii
+- Ed is traveling back from Japan
+
+## 🃏 Joke of the Day
+
+Did you know that there's a crocodile that can jump higher than the average house?
+This is due to its powerful back legs and the fact that the average house can't jump!
```
Comment (LoganParker) [inline docs/meetings/team/2023_02_27.md:25]: This is really good, digging the Florida pun themes

**Review by LoganParker — APPROVED** [review]
Minutes look great, thanks Veronica!

**Pull request merged**